tests/execution.rs

```diff
@@ -2684,6 +2692,11 @@ fn test_err_instruction_count_syscall_capped() {
 
 #[test]
 fn test_non_terminate_early() {
+    let config = Config {
+        enabled_sbpf_versions: SBPFVersion::V1..=SBPFVersion::V1,
```

Comment:
Instead of limiting this to SBPF-v1 we should change the exception thrown to e.g. jump out of bounds.

Reply:
Jump out of bounds, and call to an invalid offset are all detected during verification, so this test wouldn't reach the execution stage.

Reply:
Going to do `call r6`, instead of the sys call.